src/components/form/Form.js

```diff
@@ -48,10 +80,93 @@ const Form = () => {
             Find Venues
           </button>
         </section>
-        { formCounter > 0 && (<FormVenue 
-          incrementForm={incrementForm}
-        />)}
-        { formCounter > 1 && <FormTime incrementForm={incrementForm}/> }
+        { formCounter > 0 && (
+          <section className='form-section'>
+            <select value={selectedVenue} onChange={(event) => handleVenueSelect(event)}>
+              <option value={''}>Please Select A Venue</option>
+              <option value={'newVenue'}>Add New Venue</option>
+              {findVenues()}
+            </select>
```

Comment:
Im glad we finally sorted this out! That was a confusing challenge.

Reply:
Wow! Y'all did some work to make this so concise!